schemas/v3.1/schema.yaml

```diff
@@ -0,0 +1,836 @@
+              servers:
+                type: array
+                items:
+                  $ref: 'Server'
+              parameters:
+                type: array
+                items:
+                  $
+        unevaluatedProperties: false
+
+      Operation:
+        $id: '#Operation'
+        type: object
+        required:
+          - responses
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Deprecatable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          tags:
+            type: array
+            items:
+              type: string
+          summary:
+            type: string
+          externalDocs:
+            $ref: '#ExternalDocumentation'
+          operationId:
+            type: string
+          parameters:
+            type: array
+            items:
+              $ref: '#Parameter'
+          requestBody:
+            $ref: '#RequestBody'
+          responses:
+            $ref: '#Responses'
+          callback:
+            $ref: '#Callback'
+          security:
+            type: array
+            items:
+              $ref: '#SecurityRequirement'
+          servers:
+            type: array
+            items:
+              $ref: '#Server'
+        unevaluatedProperties: false
+
+      Responses:
+        $id: '#Responses'
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          default:
+            $ref: '#Response'
+        patternProperties:
+          '[1-5](?:\d{2}|XX)':
+            $ref: '#Response'
+        minProperties: 1
+        unevaluatedProperties: false
+
+      SecurityRequirement:
+        $id: '#SecurityRequirement'
+        type: object
+        additionalProperties:
+          type: array
+          items:
+            type: string
+
+      Tag:
+        $id: '#Tag'
+        type: object
+        required:
+          - name
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          name:
+            type: string
+          externalDocs:
+            $ref: '#ExternalDocumentation'
+        unevaluatedProperties: false
+
+      Parameter:
+        $id: '#Parameter'
+        oneOf:
+          - $ref: '#Reference'
+          - type: object
+            allOf:
+              - $ref: '#/$defs/Mixins/$defs/WithNameAndLocation'
+              - $ref: '#/$defs/Mixins/$defs/Extensible'
+              - oneOf:
+                  - allOf:
+                      - $ref: '#PathParam'
+                      - $ref: '#/$defs/Mixins/$defs/Describable'
+                      - $ref: '#/$defs/Mixins/$defs/Deprecatable'
+                      - required:
+                          - required
+                        properties:
+                          required:
+                            const: true
+                            default: true
```

Comment:
You may want to move this default up to the object level, as `default: { required: true }`, depending on how it is intended to be used (by implementations, as an annotation, rather than informatively by a human).

Reply:
I recommend leaving it here.